jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/URIUtil.java

```diff
@@ -1845,10 +1847,52 @@ public static URI toURI(String resource)
     {
         Objects.requireNonNull(resource);
 
-        // Only try URI for string for known schemes, otherwise assume it is a Path
-        return (ResourceFactory.isSupported(resource))
-            ? correctFileURI(URI.create(resource))
-            : Paths.get(resource).toUri();
+        if (URIUtil.hasScheme(resource))
+        {
+            try
+            {
+                URI uri = new URI(resource);
+
+                if (ResourceFactory.isSupported(uri))
+                    return correctFileURI(uri);
+
+                // We don't have a supported URI scheme
+                if (uri.getScheme().length() == 1)
+                {
+                    // Input is a possible Windows path disguised as a URI "D:/path/to/resource.txt".
+                    try
+                    {
+                        return toURI(Paths.get(resource).toUri().toASCIIString());
+                    }
+                    catch (InvalidPathException ignore)
+                    {
+                        // ignore
```

Comment:
We never really came up with a good standard replacement of ignored exceptions. Should we:
```suggestion
                        LOG.trace("ignored", ignore);
```